### PR TITLE
WIP: Fix package page

### PR DESCRIPTION
**Pull request by xbpcb**
Almost done.
Line Chart  groupBy seems incorrect, need to check.
Everything must be checked again.

### CODE REVIEW

**Comment by MartinKolarik** [comment]
- [ ] dropdown style (was correct before)
image

**Comment by xbpcb** [comment]
Line Chart groupBy fixed

**Comment by xbpcb** [comment]
I will replace deprecated endpoints in this branch, since most of them related to Package page and it's components

**Comment by MartinKolarik** [comment]
- [ ] also version dropdown

**Comment by MartinKolarik** [comment]
- [x]  when displayed on package and not in search results, these should have no hover effect and regular cursor
 - [ ] ^ actually applies to the popup search too because we don't have the filtering there
image

File: src/views/components/package-sidemenu.html [hunk 1]
```diff
@@ -155,49 +155,36 @@
...
-								this.animate('bandwidthTotal', _.convertBytesToUnits(response.total), { duration: 1000 });
+								this.set('packageBandwidthStats', response.bandwidth);
+								this.animate('bandwidthTotal', _.convertBytesToUnits(response.bandwidth.total), { duration: 1000 });
 							} else {
 								this.set('noDataOfBandwidth', true);
 							}
 						});
+
+						// get package top version stats
+						http.fetchPackageVersionsStats(type, name, period, 'hits', 1).then((response) => {
```
Comment (MartinKolarik) [inline src/views/components/package-sidemenu.html:180]: Let's use the same limit everywhere to take advantage of browser caching.

**Comment by MartinKolarik** [comment]
I'm going to merge this, please address the remaining points separately:

- use real download numbers in "show/configure files"
- Open http://localhost:4400/package/npm/slick-carousel, then click "statistics" tab. The chart gets weirdly animated from left to right (only happens on the first tab switch)

**Pull request merged**